docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -136,21 +137,21 @@ deviceCommands:
 
 ### Mount the custom-config
 
-Open the `edgex-compose/compose-builder/docker-compose.yml` file and then add volumes path and environment as shown below:
+Create a docker-compose file `docker-compose.override.yml` for adding volumes path and override config files as shown below:
```

Comment:
add a reference to introduce what docker-compose override is

Reply:
Added the reference link.